src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
Field names should really be lowercase. I think this should just be initialized to the default value of PlatformAxoloti as well so we can avoid checking for null and empty later.